pages/blocks/video/video.js

```diff
@@ -31,7 +32,10 @@ export default async function decorate(block) {
       wrapper.innerHTML = '<div class="video-placeholder-play"><button title="Play"></button></div>';
       wrapper.prepend(pic);
       wrapper.addEventListener('click', () => {
-        renderVideo(block, source, true);
+        requestAnimationFrame(() => {
+          block.setAttribute('style', `height:${wrapper.offsetHeight}px`);
```

Comment:
I think you may also need to set the block as position relative here, and in the css, make the `video.centered` element position absolute. at least for me, the centering doesn't appear to work as it is currently.

Reply:
Sorry what is it you're seeing, not vertically or horizontally centered?

Reply:
![538A8E9D-AE0E-43D8-8EB9-6CD807A168E5](https://github.com/hlxsites/pgatour-com/assets/84591646/b8a6d808-f769-44ef-ac11-8725fbaa5f21)

Reply:
not vertically centered, this on desktop safari.

before clicking play
<img width="1410" alt="image" src="https://github.com/hlxsites/pgatour-com/assets/1646396/c51994c4-159f-492a-8025-b37ded23e08a">


after clicking play

<img width="1340" alt="image" src="https://github.com/hlxsites/pgatour-com/assets/1646396/effa7814-8c9c-4f54-bb41-a97ae892ba8b">

Reply:
Thanks, hadn't tested Safari.

This has been resolved.